tests/format/FunctionCalls/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -80,7 +80,8 @@ contract FunctionCalls {
                     keccak256(req.data)
                 )
             )
-        )._hashTypedDataV2(
+        )
+        ._hashTypedDataV2(
```

Comment:
This is definitely an improvement, but FWIW prettier core would've indented this and the rest of the chain (which makes sense to me).

No need to change this in this PR, we can open a new issue for that.

Reply:
Are you sure? I did cross check with prettier when I was designing the logic behind this.
I'll merge and we can give it another review later

Reply:
Wow I had another check and indeed it does indent. This really simplifies a lot for this behaviour then 🤣 Let's open an issue for this though.